### DISCUSSION TITLE
What Blog Topics are Allowed in Github Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22909] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, Can we write blog posts related to dentistry, law, finance, health issues, weight loss, art, music, fashion, pets etc. etc. on blogs hosted on Github pages or only coding and tech related blogs are allowed on Github pages?</p>
<p>I want to create my personal rambling blog.</p>
<p>Please let me know.</p>
<p>Thank you.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/devoidd">@devoidd</a>,</p>
<p>As long as your blog doesn’t violate  <a href="https://help.github.com/articles/github-terms-of-service/" rel="nofollow noopener">GitHub’s Terms of Service</a>, you can write about any topic. I personally have a blog on hydroponics that I was working on a couple of years back and hosted on GitHub Pages. People are more than their code, so I hope you have fun with your blog and make it about whatever interests you. :slight_smile:</p>
<p>Thanks!</p>

**Comment by ghost** [comment]
<p>Wow, Thank you That Pat :slight_smile:</p>
<p>I am also into gardening but not into Hydroponics.</p>

**Comment by ThirdEyeRose** [comment]
<p>You’re very welcome!</p>
